vue/sbc-common-components/public/index.html

```diff
@@ -5,13 +5,18 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width,initial-scale=1.0">
     <link rel="icon" href="<%= BASE_URL %>favicon.ico">
+    <link rel="stylesheet" href="<%= BASE_URL %>css/addresscomplete-2.30.css">
+    <script type="text/javascript" src="<%= BASE_URL %>js/addresscomplete-2.30.js" defer></script>
```

Comment:
PS I copied/cached the css and js files locally so we never hang on startup if the Canada Post website is slow/down.

Reply:
Pretty cool! 
Does caching like this have any impact on load times etc. 
I imagine in this case, the content is probably relatively small but for examples sake, would doing something like this with a large library have an impact?

Reply:
Of course it has an impact, but I assume it's faster to load from our server than an external server (esp if the external server is down).
Note also that I use the "defer" prop to load the script as late as possible in the app loading flow.

Reply:
if render blocking is the issue , should async wont work?
https://developers.google.com/speed/docs/insights/BlockingJS

Reply:
It might work but it's deprecated 😛

Reply:
what is deprecated? async attirbute?

Reply:
I just looked again. Ooops, API is deprecated not "async".

I believe "defer" provides faster initial loading than "async" here.

Reply:
hm.. did it still block the rendering [slowed the page loading] , when used async?

Reply:
but Thanks for taking this up and pushing this change..I am just curious why async is still blocking the rendering

Reply:
I don't remember my test results when I initially tried async vs defer and decided on the latter.

(Would async load scripts at the same time? If so then this would reduce the bandwidth available for other loads.)

(Also I think defer doesn't autorun the script until everything else is loaded, which is desired here -- there is no need to run the AddressComplete init method until after the app is loaded.)